index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="Sample Microverse Hello Project">
+        <meta name="author" content="Albert Mensah-Ansah">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+        <title>My Portfolio</title>
+    </head>
+    <body>
+
+        <nav class="topnav">
+            <div class="welcome">Welcome</div>
```

Comment:
Kindly your logo should be inside `a` tag, so anyone clicks on it switch him to the index page.
`<a herf="index.html" class="welcome">Welcome</a>`